reference/checkouts.v3.yml

```diff
@@ -8619,7 +8622,7 @@ components:
       items:
         type: object
         properties:
-          address:
+          shipping_address:
```

Comment:
I remember we allow both `address` and `shipping_address` (we defined them as alias). am I right? @valentindellangela

Reply:
![image](https://github.com/bigcommerce/api-specs/assets/94653751/905485ef-836b-4889-93c8-bdbb99857d41)

`shipping_address` (deprecated) or `address`

This is from storefront doc description. I checked the s2s part, it should be the same. and we encourage to use address